entrypoint.sh

```diff
@@ -18,12 +18,12 @@ publish_dependencies_as_layer(){
 publish_function_code(){
 	echo "Deploying the code itself..."
 	zip -r code.zip . -x \*.git\*
-	aws lambda update-function-code --function-name "${LAMBDA_FUNCTION_NAME}" --zip-file fileb://code.zip
+	aws lambda update-function-code --function-name "${INPUT_LAMBDA_FUNCTION_NAME:-$LAMBDA_FUNCTION_NAME}" --zip-file fileb://code.zip
 }
 
 update_function_layers(){
 	echo "Using the layer in the function..."
-	aws lambda update-function-configuration --function-name "${LAMBDA_FUNCTION_NAME}" --layers "${LAMBDA_LAYER_ARN}:${LAYER_VERSION}"
+	aws lambda update-function-configuration --function-name "${INPUT_LAMBDA_FUNCTION_NAME:-$LAMBDA_FUNCTION_NAME}" --layers "${INPUT_LAMBDA_LAYER_ARN:-$LAMBDA_LAYER_ARN}:${LAYER_VERSION}"
```

Comment:
Should `LAYER_VERSION` also be included in this change? I think we can also forgo the whole falling back on env variables thing and make them required parameters for a cleaner implementation of v2. What do you think?